site/content/docs/5.3/components/stepped-process.md

```diff
@@ -16,6 +16,10 @@ Use short labels for each step, otherwise they will be cut off to preserve inlin
 
 Add `.active` to a `.stepped-process-item` to indicate the current step, alongside `aria-current="step"` to convey the active state to assistive technologies.
 
+{{< callout info >}}
+**Number of steps required** Stepped process requires the total number of steps for CSS calculation. We do this with the `$stepped-process-max-items` variable. You can easily customize this variable in your projects.
```

Comment:
```suggestion
**Using more than 5 items ?** You can easily customize this by changing `$stepped-process-max-items` variable before importing `mixins`.
```

Reply:
Ok for your version, it sounds less 'strict'.